src/content/contribute/writing-a-loader.md

```diff
@@ -234,13 +234,15 @@ __src/loader.js__
 ``` js
 import { getOptions } from 'loader-utils';
 
-export default source => {
+function loader(source) {
```

Comment:
you could do `export default function loader(...) {` to keep the code consistent

Reply:
You're right, updated!